app/src/main/java/org/mifos/mobile/ui/about/AboutUsScreen.kt

```diff
@@ -0,0 +1,68 @@
+package org.mifos.mobile.ui.about
+
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.material3.*
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.lifecycle.viewmodel.compose.viewModel
+import org.mifos.mobile.core.ui.component.AboutUsItemCard
+import org.mifos.mobile.core.ui.component.MifosItemCard
+import org.mifos.mobile.ui.enums.AboutUsListItemId
+import java.util.*
+
+@SuppressWarnings("UnusedMaterial3ScaffoldPaddingParameter")
+@Composable
+fun AboutUsScreen(viewModel: AboutUsViewModel = viewModel()) {
+    Column(
+        modifier = Modifier
+            .fillMaxSize()
+            .padding(16.dp, 56.dp, 16.dp, 16.dp)
```

Comment:
You can remove this `56dp` and use one padding `16dp`, and you can use the Spacer widget to add space

Reply:
I will try to change it but the xml version had 56 dp as the padding and thats what i have used here

Reply:
done